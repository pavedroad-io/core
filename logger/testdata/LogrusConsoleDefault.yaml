logpackage: logrus
loglevel: info
enabletimestamps: false
enablecolorlevels: true
enablecloudevents: true
enablekafka: false
kafkaformat: cloudevents
kafkaproducercfg:
  brokers:
  - localhost:9092
  topic: logs
  partition: random
  key: fixed
  keyname: user
  cloudeventsid: hmac
  compression: snappy
  ackwait: local
  flushfreq: 500ns
  enabletls: false
  tlscfg: null
  enabledebug: false
enableconsole: true
consoleformat: text
enablefile: false
fileformat: json
filelocation: pavedroad.log
