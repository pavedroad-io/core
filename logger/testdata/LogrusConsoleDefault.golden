[36mINFO[0m Infof using logrus                            [36msource[0m="http://github.com/pavedroad-io/core/go/logger" [36mspecversion[0m=1.0 [36mtype[0m=io.pavedroad.cloudevents.log
[33mWARN[0m Warnf using logrus                            [33msource[0m="http://github.com/pavedroad-io/core/go/logger" [33mspecversion[0m=1.0 [33mtype[0m=io.pavedroad.cloudevents.log
[31mERRO[0m Errorf using logrus                           [31msource[0m="http://github.com/pavedroad-io/core/go/logger" [31mspecversion[0m=1.0 [31mtype[0m=io.pavedroad.cloudevents.log
[36mINFO[0m Print usinglogrus                             [36msource[0m="http://github.com/pavedroad-io/core/go/logger" [36mspecversion[0m=1.0 [36mtype[0m=io.pavedroad.cloudevents.log
[36mINFO[0m Printf using logrus                           [36msource[0m="http://github.com/pavedroad-io/core/go/logger" [36mspecversion[0m=1.0 [36mtype[0m=io.pavedroad.cloudevents.log
[36mINFO[0m Println using logrus                          [36msource[0m="http://github.com/pavedroad-io/core/go/logger" [36mspecversion[0m=1.0 [36mtype[0m=io.pavedroad.cloudevents.log
